test/e2e/operator_test.go

```diff
@@ -228,7 +246,7 @@ var _ = Describe("Operator API", func() {
 			return err
 		}).Should(Succeed())
 
-		// Ensure that o is consistently not found
+		By("Ensure that o is consistently not found")
```

Comment:
This one already has a `By`. Reduce to 1?

Reply:
Looks like a few others as well.

Reply:
If it's worth calling out in a comment, it's worth calling out in a `By()`.

Reply:
Doesn't ginkgo group code from one `By` to the next in some way? I think with two consecutive `By` lines, you're ending up with empty groupings? Not positive on this since I've never seen two consecutive `By`s though.

Reply:
Yes, but only if you append a closure to the items in `By()`. This is a no-op informative log.